bundles/org.openhab.binding.neato/src/main/java/org/openhab/binding/neato/internal/NeatoRobot.java

```diff
@@ -146,21 +144,13 @@ public void sendCommand(String command) throws NeatoCommunicationException {
 
                 request.addParam("mode", this.state.getCleaning().getModeValue());
                 request.addParam("category", HOUSE.getCategory());
-
-                Integer navigationMode = this.state.getCleaning().getNavigationModeValue();
-                if (Integer.valueOf(TURBO.getMode()).equals(this.state.getCleaning().getModeValue())) {
-                    // From the Neato API Docs...
-                    // Note that navigationMode can only be set to 3 if mode is 2,
-                    // otherwise an error will be returned.
-                    navigationMode = DEEP.getNavigationMode();
```

Comment:
What model robot have you tested this against?  I remember we had to put in this block specifically because of errors we were seeing.  I can deploy this change and test against my D7 again

Reply:
I did not test it, but I figured out via a comment on twitter that going into DEEP mode via the API caused the same weird behaviour. I use the D5. See https://twitter.com/dlfranks/status/1390048438457409540

Reply:
How can I test it?

Reply:
You can build this branch and take the neato binding jar and drop it into your openhab installation.  See https://www.openhab.org/docs/configuration/addons.html#through-manually-provided-add-ons

Reply:
I think we need to keep this conditional check in place since the Neato API requires deep navigation mode if cleaning mode is set to turbo.  

The other change to echo the current cleaning and navigation modes should be fine.

Reply:
@jjlauterbach I tried to build the repo, but I was unable to do so. 

When set to ECO, my D5 acts normal now. So I think the navigationMode may be model specific? 

Someone has a related problem: https://github.com/openhab/openhab-addons/issues/9917

Reply:
Okay - I will build and deploy this locally today to verify

Reply:
I was able to test things and every thing is working fine on my D7.  I double checked the API docs again and I think they were misinterpreted originally and our safe guard logic was a bit backwards.  It should have been to ensure mode is set to TURBO when navigation mode is DEEP.  DEEP navigation is not something that can be set from the Neato app (from what I've seen) so we can probably just safely ignore this.